tests/foreman/cli/test_import.py

```diff
@@ -315,14 +341,56 @@ def test_import_users_default(self, data):
         })
         os.remove(new_dataset)
         ssh.command(
-            'rm -rf ${{HOME}}/.transition_data {0} {1}'
-            .format(pwdfile, tmp_dir)
+            'rm -rf ${{HOME}}/.transition_data {}'
+            .format(pwdfile)
         )
         self.assertEqual(
             ssh_import.stdout,
             [u'Summary', u'  Created 3 users.', u'']
         )
 
+    @data(*positive_import_user_data())
+    def test_reimport_users_default(self, data):
```

Comment:
Ugh, my bad.
the `data` variable has to be used, I just forgot to replace the call to `*positive_import_*_data()`  by the variable in all tests. Good catch - fixed.